package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@heetch/flamingo-react",
-  "version": "1.9.3",
+  "version": "2.0.0",
```

Comment:
Nope nope nope, let Lerna handle that 🙂 
Also, we should release a RC before the v2

Reply:
ops. I was trying out if lerna worked. Will change